nucypher/blockchain/eth/agents.py

```diff
@@ -185,6 +187,14 @@ def get_current_period(self) -> int:
     # StakingEscrow Contract API
     #
 
+    def get_all_locked_tokens(self, periods: int = 1) -> int:
```

Comment:
Why `periods=1` is the default and not 0, like `get_locked_tokens()`?

Reply:
There is a slight misunderstanding in naming conventions here.  The signatures of `getAllLockedTokens` and `getLockedTokens` are similar but behave slightly differently internally.  See:  <https://github.com/nucypher/nucypher/blob/master/nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol#L313>

Reply:
Thanks for the clarification!